ziti/cmd/create/create_config_controller.go

```diff
@@ -150,7 +149,6 @@ func NewCmdCreateConfigController() *CreateControllerConfigCmd {
 
 func (options *CreateConfigControllerOptions) addFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&options.CtrlPort, optionCtrlPort, constants.DefaultCtrlAdvertisedPort, "port used for the router to controller communication")
-	cmd.Flags().StringVar(&options.DatabaseFile, optionDatabaseFile, "ctrl.db", "location of the database file")
```

Comment:
Yes. This removes an incomplete implementation that was never in effect. The command-line option had been added, but the template never respected the argument because it was hard-coded with the default value.

Reply:
we can never be sure someone didn't/doesn't use it for whatever reason. it'll still need a "we removed this and replaced it with an env var" release note

Reply:
@dovholuknf The implementation that I removed here was incomplete, and so there's no possibility that it was ever used. The option didn't do anything at all.